packages/ketcher-react/src/script/ui/data/convert/structConverter.ts

```diff
@@ -26,6 +27,28 @@ export function couldBeSaved(
 ): string | null {
   let warnings: Array<string> = []
   const formatName: string = getPropertiesByFormat(format).name
+
+  const rxnArrowsSize = struct.rxnArrows.size
+  const hasRxnArrow = struct.hasRxnArrow()
+
+  if (format !== 'graph') {
+    if (hasRxnArrow) {
+      const rxnArrowMode: RxnArrowMode | undefined = struct.rxnArrows.get(0)?.mode 
```

Comment:
You can use 'struct.rxnArrows.get(0)!' instead (you have already checked that  it exists before - struct.hasRxnArrow() function)

Reply:
Fixed